.github/workflows/pr_views_to_core.yml

```diff
@@ -0,0 +1,38 @@
+name: PR Views to PrefectHQ/prefect
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "views/aggregate-worker-metadata.json"
+  workflow_dispatch:
+
+env:
+  GITHUB_TOKEN: ${{ secrets.OSS_PRS_RW }}
+
+jobs:
+  update-views:
+    name: Send updated views to PrefectHQ/prefect
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Open PRs to PrefectHQ/prefect
+        run: |
+          NEW_BRANCH="update-worker-metadata-$(date +%Y%m%d%H%M%S)"
+          TARGET_FILE_PATH="src/prefect/server/api/collections_data/views/aggregate-worker-metadata.json"
+          SOURCE_REPO_URL="https://raw.githubusercontent.com/PrefectHQ/prefect-collection-registry/main/views/aggregate-worker-metadata.json"
+
+          git clone https://github.com/PrefectHQ/prefect.git
+          cd prefect
+
+          mkdir -p $(dirname $TARGET_FILE_PATH)
```

Comment:
just curious for my own sake - why is `mkdir` needed (since you already cloned the repo)?

Reply:
hmm I guess if we assume that the `TARGET_FILE_PATH` definitely exists in the cloned repo then this isn't necessary, which I think is actually a fine assumption - can update

Reply:
updated in https://github.com/PrefectHQ/prefect-collection-registry/pull/335/commits/ecc454167be0775ff4d0f458cd20a8ccf5a8e835